src/components/filter/criteriaBased/CriteriaBasedForm.tsx

```diff
@@ -58,19 +79,26 @@ function CriteriaBasedForm({ equipments, defaultValues }: CriteriaBasedFormProps
                     message="changeTypeMessage"
                     validateButtonLabel="button.changeType"
                 />
-            </Grid>
-            {watchEquipmentType &&
-                equipments[watchEquipmentType] &&
-                equipments[watchEquipmentType].fields.map((equipment: any, index: number) => {
-                    const EquipmentForm = equipment.renderer;
-                    const uniqueKey = `${watchEquipmentType}-${index}`;
-                    return (
-                        <Grid item xs={12} key={uniqueKey} flexGrow={1}>
-                            <EquipmentForm {...equipment.props} />
-                        </Grid>
-                    );
-                })}
-        </Grid>
+            </Box>
+            <Box sx={styles.ScrollableContainer}>
+                <Box sx={styles.ScrollableContent}>
+                    <Grid container spacing={2}>
+                        {watchEquipmentType &&
+                            equipments[watchEquipmentType] &&
+                            equipments[watchEquipmentType].fields.map((equipment: any, index: number) => {
```

Comment:
any is not accepted

Reply:
we can profit this PR to declare equipment here as FormField

Reply:
Ok, typed s a FormField0